discovery/lib/builder/builder_test.go

```diff
@@ -0,0 +1,153 @@
+package builder
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+)
+
+func TestGjson(t *testing.T) {
```

Comment:
Is leaving this test here worth it? It lets us catch behavioral changes in the packages and acts as an example, but it's still atypical to test external packages.